vscode/test/suites/debugger/debugger.test.ts

```diff
@@ -0,0 +1,567 @@
+      name: "Launch foo.qs",
+      type: "qsharp",
+      request: "launch",
+      program: "${workspaceFolder}src/foo.qs",
+      stopOnEntry: true,
+    });
+
+    // should break on entry (per debug config above)
+    await waitUntilPaused([
+      {
+        id: 0,
+        source: {
+          name: "foo.qs",
+          path: "vscode-test-web://mount/src/foo.qs",
+          sourceReference: 0,
+          adapterData: "qsharp-adapter-data",
+        },
+        line: 5,
+        column: 9,
+        name: "Foo ",
+        endLine: 5,
+        endColumn: 15,
+      },
+      { id: 0, line: 0, column: 0, name: "entry", source: undefined },
+    ]);
+
+    // step into call (will be a call into bar.qs)
+    await vscode.commands.executeCommand("workbench.action.debug.stepInto");
+
+    await waitUntilPaused([
+      {
+        id: 1,
+        source: {
+          name: "bar.qs",
+          path: "vscode-test-web://mount/src/bar.qs",
+          sourceReference: 0,
+          adapterData: "qsharp-adapter-data",
+        },
+        line: 3,
+        column: 9,
+        name: "Bar ",
+        endLine: 3,
+        endColumn: 26,
+      },
+      {
+        id: 0,
+        source: {
+          name: "foo.qs",
+          path: "vscode-test-web://mount/src/foo.qs",
+          sourceReference: 0,
+          adapterData: "qsharp-adapter-data",
+        },
+        line: 5,
+        column: 12,
+        name: "Foo ",
+        endLine: 5,
+        endColumn: 14,
+      },
+      { id: 0, line: 0, column: 0, name: "entry", source: undefined },
+    ]);
+
+    // text editor should now be open on bar.qs
+    await waitForTextEditorOn(barUri);
+  });
+
+  test("Step into standard lib", async () => {
+    // Set a breakpoint on line 8 of foo.qs (7 when 0-indexed)
+    // This will be a call into stdlib
+    await vscode.debug.addBreakpoints([
+      new vscode.SourceBreakpoint(
+        new vscode.Location(fooUri, new vscode.Position(7, 0)),
+      ),
+    ]);
+
+    // launch debugger
+    await vscode.debug.startDebugging(workspaceFolder, {
+      name: "Launch foo.qs",
+      type: "qsharp",
+      request: "launch",
+      program: "${workspaceFolder}src/foo.qs",
+      stopOnEntry: false,
+    });
+
+    // should hit the breakpoint we set above
+    await waitUntilPaused([
+      {
+        id: 0,
+        source: {
+          name: "foo.qs",
+          path: "vscode-test-web://mount/src/foo.qs",
+          sourceReference: 0,
+          adapterData: "qsharp-adapter-data",
+        },
+        line: 8,
+        column: 9,
+        name: "Foo ",
+        endLine: 8,
+        endColumn: 14,
+      },
+      { id: 0, line: 0, column: 0, name: "entry", source: undefined },
+    ]);
+
+    // step into call (will be a call into intrinsic.qs)
+    await vscode.commands.executeCommand("workbench.action.debug.stepInto");
+
+    await waitUntilPaused([
+      {
+        id: 1,
+        source: {
+          name: "qsharp-library-source:intrinsic.qs",
```

Comment:
Nice! I see we have a test for stepping into the stdlib, so maybe ignore my last comment :-)